api/RTCRtpContributingSource.json

```diff
@@ -80,26 +80,38 @@
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": {
-              "version_added": "59",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "media.peerconnection.rtpsourcesapi.enable",
-                  "value_to_set": "true"
-                }
-              ]
-            },
-            "firefox_android": {
-              "version_added": "59",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "media.peerconnection.rtpsourcesapi.enable",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": "59",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.peerconnection.rtpsourcesapi.enable",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "60",
+                "notes": "Starting in Firefox 60, the <code>timestamp</code> is correctly computed based on the window's <code>Performance</code> time, rather than <code>Date.getTime()</code>."
```

Comment:
This should be a note of the previous block (the one with "version_added": "60".